### DISCUSSION TITLE
Inform author when post is deleted

### DISCUSSION THREAD

**Discussion by enno123** [NamelessMC/Nameless#2139] [Ideas]
I'm missing a feature in this forum. If I delete a post, I want to inform the author why the post was removed. When the post is deleted, another staff should be able to see the post but it should be highlighted in red, so they can see, that the post was once removed. They should also be able to see, why the post was removed. The User should only be able to see the reason and the staff who deleted it.

**Comment by tadhgboyle** [comment]
Great idea!
